### DISCUSSION TITLE
Disable Success Descriptions

### DISCUSSION THREAD

**Discussion by MaiFlow** [Jagusti/fvtt-wfrp4e-gmtoolkit#210] [Q&A] [answered]
This module is great, and exactly what I need to handle my needs during play. One very minor irritant for me however is how enabling it changes rolls so that they add the success descriptions from the book to rolls.

Is there a way to disable this so that roll templates return to how they work in the core system? 

Basically, I want rolls to look like this:
✔️ **Description:** Success
Not like this:
❌ **Description:** Success. 'Yes'

**Comment by Jagusti** [comment, marked as answer]
Glad you're enjoying the Toolkit. Thanks for sharing your question here.

Short answer is no, the extended descriptions can't be disabled. 

The reason is that it's an override on the system descriptions that is loaded when translation files are compiled as Foundry cranks up, rather than a configured or computed setting they you can access or trigger once the game works is available.  

That said, there are a couple of ways to override this  

1. You can safely delete [the relevant lines](https://github.com/Jagusti/fvtt-wfrp4e-gmtoolkit/blob/dev/lang/en.json#L302-L309) in the translation files. You would have to do that each time the module is updated though. The files are in the `Data/modules/wfrp4e-gm-toolkit/lang/` folder. They are *.json files that can be edited with a regular text editor. 

2. Or configure (in world.json) and add your own translation file (eg, en.json) for the world. The advantage here is you don't have to redo this every time the Toolkit is updated. You can also make other translation changes in here if you want to. These are the [comparable lines in module.json](https://github.com/Jagusti/fvtt-wfrp4e-gmtoolkit/blob/dev/module.json#L56-L82) for Toolkit - you can copy these into your world.json, removing any languages you don't use. 

I'm on mobile but can add more detail when I'm back at a computer if you need it.

**Reply by MaiFlow** [reply]
Thank you for the very detailed instructions; that fixed it perfectly!

I think I will go with option 1. until I get more familiar with Foundry, but it works for now. No more seeing extra text on my rolls. I am very happy!

**Comment by Jagusti** [comment]
Hmmm .... [this example](https://discord.com/channels/170995199584108546/670336275496042502/1107714154799833099) may be worth exploring. 

```js
export const i18n = {
  "Spells": null,
}

Hooks.once("init", () => {
  const variant = "Scifi";
  Object.keys(i18n)
    .forEach(key => i18n[key] = game.i18n.translations.MySystem[key][variant])
});
```

**Reply by Jagusti** [reply]
And potentially use the [i18Init hook](https://foundryvtt.com/api/modules/hookEvents.html#i18nInit) instead.
